sky/packages/sky/lib/widgets/drawer.dart

```diff
@@ -130,27 +158,27 @@ class Drawer extends AnimatedComponent {
     );
   }
 
-  double get xPosition => _position.value.x;
-
   void _onStatusChanged(AnimationStatus status) {
-    if (status == AnimationStatus.dismissed &&
-        navigator != null &&
-        navigator.currentRoute is RouteState &&
-        (navigator.currentRoute as RouteState).owner == this) // TODO(ianh): remove cast once analyzer is cleverer
-      navigator.pop();
-    if (onStatusChanged != null)
-      onStatusChanged(status);
+    scheduleMicrotask(() {
```

Comment:
navigator.pop ends up calling performance.reverse() which calls performance.fling. Calling performance.fling from within a timeline-updated callback hits an assert in Ticker._scheduleTick. (Because it ends up calling Ticker.start from within Ticker._tick).

Alternatively, I think I could fix this by only having pop() not hide the drawer when called from here.